### PR TITLE
Add a tutorial for using AWS ECR with registry-creds addon

### LINKED ISSUE #5753: Add documentation for configuring AWS Elastic Container Registry
**The exact command to reproduce the issue**:

```
minikube start
minikube addons configure registry-creds
minikube addons enable registry-creds
```
**The full output of the command that failed**:<details>

😄  minikube v1.5.0 on Darwin 10.15
✨  Automatically selected the 'hyperkit' driver (alternates: [virtualbox])
🔥  Creating hyperkit VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.16.2 on Docker 18.09.9 ...
🚜  Pulling images ...
🚀  Launching Kubernetes ... 
⌛  Waiting for: apiserver proxy etcd scheduler controller dns
🏄  Done! kubectl is now configured to use "minikube"

Do you want to enable AWS Elastic Container Registry? [y/n]: y
-- Enter AWS Access Key ID: my-access-key
-- Enter AWS Secret Access Key: my-secret-key
-- (Optional) Enter AWS Session Token:
-- Enter AWS Region: ap-southeast-1
-- Enter 12 digit AWS Account ID (Comma separated list): my-account-id
-- (Optional) Enter ARN of AWS role to assume:

Do you want to enable Google Container Registry? [y/n]: n

Do you want to enable Docker Registry? [y/n]: n
❌  ERROR creating `registry-creds-ecr` secret: Temporary Error: secrets "registry-creds-ecr" not found
❌  ERROR creating `registry-creds-gcr` secret: Temporary Error: secrets "registry-creds-gcr" not found
⚠️  ERROR creating `registry-creds-dpr` secret
✅  registry-creds was successfully configured
✅  registry-creds was successfully enabled

</details>

**The output of the `minikube logs` command**:

No logs output due to the command `minikube addons configure registry-creds`
Following are the output due to the command `minikube addons enable registry-creds`

```
Oct 28 04:05:00 minikube kubelet[2947]: I1028 04:05:00.411350    2947 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "default-token-wrmh7" (UniqueName: "kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-default-token-wrmh7") pod "registry-creds-69f86f67f7-5zxnr" (UID: "23240816-32bf-4532-946e-0e8a4fe2567e")
Oct 28 04:05:00 minikube kubelet[2947]: I1028 04:05:00.413010    2947 reconciler.go:207] operationExecutor.VerifyControllerAttachedVolume started for volume "gcr-creds" (UniqueName: "kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds") pod "registry-creds-69f86f67f7-5zxnr" (UID: "23240816-32bf-4532-946e-0e8a4fe2567e")
Oct 28 04:05:00 minikube kubelet[2947]: E1028 04:05:00.513737    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:00 minikube kubelet[2947]: E1028 04:05:00.517650    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:01.017604162 +0000 UTC m=+1099.455026157 (durationBeforeRetry 500ms). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
Oct 28 04:05:01 minikube kubelet[2947]: E1028 04:05:01.117915    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:01 minikube kubelet[2947]: E1028 04:05:01.118015    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:02.117992994 +0000 UTC m=+1100.555414905 (durationBeforeRetry 1s). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
Oct 28 04:05:02 minikube kubelet[2947]: E1028 04:05:02.127740    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:02 minikube kubelet[2947]: E1028 04:05:02.127938    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:04.127903919 +0000 UTC m=+1102.565325851 (durationBeforeRetry 2s). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
Oct 28 04:05:04 minikube kubelet[2947]: E1028 04:05:04.148766    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:04 minikube kubelet[2947]: E1028 04:05:04.149954    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:08.149925491 +0000 UTC m=+1106.587347430 (durationBeforeRetry 4s). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
Oct 28 04:05:08 minikube kubelet[2947]: E1028 04:05:08.178367    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:08 minikube kubelet[2947]: E1028 04:05:08.179466    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:16.17943591 +0000 UTC m=+1114.616857824 (durationBeforeRetry 8s). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
Oct 28 04:05:16 minikube kubelet[2947]: E1028 04:05:16.244690    2947 secret.go:198] Couldn't get secret kube-system/registry-creds-gcr: secret "registry-creds-gcr" not found
Oct 28 04:05:16 minikube kubelet[2947]: E1028 04:05:16.245514    2947 nestedpendingoperations.go:270] Operation for "\"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\" (\"23240816-32bf-4532-946e-0e8a4fe2567e\")" failed. No retries permitted until 2019-10-28 04:05:32.245480191 +0000 UTC m=+1130.682902111 (durationBeforeRetry 16s). Error: "MountVolume.SetUp failed for volume \"gcr-creds\" (UniqueName: \"kubernetes.io/secret/23240816-32bf-4532-946e-0e8a4fe2567e-gcr-creds\") pod \"registry-creds-69f86f67f7-5zxnr\" (UID: \"23240816-32bf-4532-946e-0e8a4fe2567e\") : secret \"registry-creds-gcr\" not found"
```

**The operating system version**:

macOS Catalina 10.15
minikube version: v1.5.0
commit: d1151d93385a70c5a03775e166e94067791fe2d9

### PR DESCRIPTION
**Pull request by prasadkatti**
fixes #5753 

This tutorial describes how one can use the `registry-creds` addon to configure credentials for AWS ECR in order to be able to pull images onto the minukube cluster.

I have used this PR to learn more about the `registry-creds` addon. So please take note of that while reviewing. :) 

P.S. Is there a linter for markdown? Do I need to add a link to this page on any other (index) page?

### CODE REVIEW

File: site/content/en/docs/Tutorials/configuring_creds_for_aws_ecr.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+- Check if the image path is valid.
+- Check if the registry-creds addon is enabled by using `minikube addons list`.
+
+## Caveats
+
+The service account token for the `default` service account in the `default` namespace is kept updated by the addon. If you create your deployment in a different namespace, the image pull will not work.
+
+## Related articles
+
+- [registry-creds addon](https://github.com/kubernetes/minikube/tree/master/deploy/addons/registry-creds)
```
Comment (medyagh) [inline site/content/en/docs/Tutorials/configuring_creds_for_aws_ecr.md]: this link seem to be broken.

Comment (prasadkatti) [reply site/content/en/docs/Tutorials/configuring_creds_for_aws_ecr.md]: Should the link point here - https://github.com/upmc-enterprises/registry-creds ? That is where the link for Registry Credentials on the addons page (https://minikube.sigs.k8s.io/docs/tasks/addons/) points to.

Comment (prasadkatti) [reply site/content/en/docs/Tutorials/configuring_creds_for_aws_ecr.md]: I opened the netlify preview link and the current link seems to be working for me.

**Comment by prasadkatti** [comment]
@medyagh - The link seems to work for me. Can you please check one more time?

**Comment by medyagh** [comment]
@prasadkatti looks good to me, thank you for adding this. look forward to see more of your contributions

**Pull request merged**